pkg/storage/minio/minio.go

```diff
@@ -147,22 +147,25 @@ func (c *Client) ScrapeArtefacts(id, directory string) error {
 	if err != nil {
 		return fmt.Errorf("failed to create a bucket %s: %w", id, err)
 	}
-	err = filepath.Walk(directory,
-		func(path string, info os.FileInfo, err error) error {
-			if err != nil {
-				return err
-			}
 
-			if !info.IsDir() {
-				err = c.SaveFile(id, path) //The function will detect if there is a subdirectory and store accordingly
+	for _, directory := range directories {
+		err = filepath.Walk(directory,
+			func(path string, info os.FileInfo, err error) error {
 				if err != nil {
 					return err
 				}
-			}
-			return nil
-		})
-	if err != nil {
-		return err
+
+				if !info.IsDir() {
+					err = c.SaveFile(id, path) //The function will detect if there is a subdirectory and store accordingly
```

Comment:
This will store the file in the bucket's subdirectory i was thinking that we maybe leave only videos/{filename} and screenshots/{filename}
Right now this will produce this:
![image](https://user-images.githubusercontent.com/8327751/142616674-566f4f46-bd54-41bd-8677-99511506b0dd.png)